include/dali/core/common.h

```diff
@@ -119,7 +119,8 @@ enum DALIImageType {
   DALI_BGR          = 1,
   DALI_GRAY         = 2,
   DALI_YCbCr        = 3,
-  DALI_ANY_DATA     = 4
+  DALI_ANY_DATA     = 4,
+  DALI_Lab          = 5
```

Comment:
I don't like DALI_ANY_DATA somewhere in the middle - neither do I think that changing it's numerical value whenever we add something is good.
@JanuszL @jantonguirao Do these values need to be sequential? 
If not, let's move DALI_ANY_DATA to some high number, so we can freely add things in between.
If the values need to be sequential, but we don't depend on DALI_RGB being 0, then maybe DALI_ANY_DATA should be at the beginning.
Moving DALI_ANY_DATA to a high number has less potential for damage, since it's rarely used.

Reply:
@mzient They don't need to be sequential. I'm OK with moving DALI_ANY_DATA to a high number

Reply:
@mzient - I'm not aware of any reason preventing us from such change, but I would change it first and double-check if that still works.

Reply:
Let's use -1 for DALI_ANY_DATA - I've checked that the relevant test works.